test/JDBC/input/temp_tables/temp_table_visibility-vu-verify.sql

```diff
@@ -0,0 +1,19 @@
+EXEC object_id_outer_proc
+go
+
+EXEC enr_list_outer_outer_proc
+go
+
+
+-- 4122 test case
+create table #t (a int)
+insert #t values(123)
+insert #t values(456)
+go
+
+if object_id('#t') is null
+    print 'fail'
+go
+
+exec babel_4122_proc
```

Comment:
perhaps im missing something obvious, but I am curious what we're testing here? We're not passing in any parameter so wont be going to the changed codepath.